pkg/controller/container-runtime-config/helpers.go

```diff
@@ -253,87 +253,67 @@ func scopeMatchesRegistry(scope, reg string) bool {
 	return false
 }
 
-// disjointOrderedMirrorSets processes icspRules and returns the resulting disjoint sets of mutual mirrors, each ordered in consistently with
-// the icspRules preference order (if possible)
-// This exists to merge different mirror sets, e.g. given mirror sets (B, C) and (A, B), it will combine them into a single (A, B, C) set.
-func disjointOrderedMirrorSets(icspRules []*apioperatorsv1alpha1.ImageContentSourcePolicy) ([][]string, error) {
-	// Collect all mirror sets into a flat array to abstract the rest of the code from the ImageContentSourcePolicy format
-	mirrorSets := []*apioperatorsv1alpha1.RepositoryDigestMirrors{}
+// mergedMirrorSets processes icspRules and returns a set of RepositoryDigestMirrors, one for each Source value,
+// ordered in consistently with the preference order of the individual entries (if possible)
```

Comment:
nit: "ordered consistently" instead of "ordered in consistently" ?

Reply:
Thanks, fixed.